src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcessDumpUtility.cs

```diff
@@ -118,30 +118,39 @@ public void TerminateProcess()
         /// <param name="filename">
         /// Filename for dump file
         /// </param>
+        /// <param name="includeFirstChanceExceptions">
+        /// Indicates whether proc dump should be configured to capture dumps on first chance exceptions.
+        /// </param>
         /// <param name="isFullDump">
         /// Is full dump enabled
         /// </param>
         /// <returns>Arguments</returns>
-        private static string BuildProcDumpArgs(int processId, string filename, bool isFullDump = false)
+        private static string BuildProcDumpArgs(int processId, string filename, bool includeFirstChanceExceptions, bool isFullDump = false)
         {
             // -accepteula: Auto accept end-user license agreement
             // -e: Write a dump when the process encounters an unhandled exception. Include the 1 to create dump on first chance exceptions.
             // -g: Run as a native debugger in a managed process (no interop).
             // -t: Write a dump when the process terminates.
             // -ma: Full dump argument.
             // -f: Filter the exceptions.
-            StringBuilder procDumpArgument = new StringBuilder("-accepteula -e 1 -g -t ");
+            StringBuilder procDumpArgument = new StringBuilder("-accepteula -g -t -e");
```

Comment:
make constants out of these parameters, & pass them to string builder.

Reply:
Readability takes a hit here.

I think the params should stay this way,